graphene-writer/src/main/kotlin/com/graphene/writer/input/kafka/deserializer/PrometheusDeserializer.kt

```diff
@@ -0,0 +1,80 @@
+package com.graphene.writer.input.kafka.deserializer
+
+import com.graphene.common.rule.GrapheneRules.SpecialChar
+import com.graphene.writer.input.GrapheneMetric
+import com.graphene.writer.input.Source
+import java.util.Objects
+import java.util.TreeMap
+import org.apache.kafka.common.serialization.Deserializer
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+class PrometheusDeserializer : Deserializer<List<GrapheneMetric>> {
+
+  private val log: Logger = LogManager.getLogger(javaClass)
+
+  override fun deserialize(topic: String?, data: ByteArray?): List<GrapheneMetric> {
+    if (Objects.isNull(data)) {
+      return emptyList()
+    }
+
+    val plainPrometheusMetrics = String(data!!).split("\n")
+
+    val grapheneMetrics = mutableListOf<GrapheneMetric>()
+    for (i in 2..plainPrometheusMetrics.size step 3) {
```

Comment:
I think the step 3 cannot be applied in general. How about check for each line and skip if the first character of a line is '#'?

Reply:
I changed the logic for supporting the case that is multiple metrics and without a timestamp.
https://github.com/graphene-monitoring/graphene/pull/48/commits/2e7e382c14736876367d1731e2f247c6230ac66b
Thanks for your comment.